src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,178 @@
+    pal_record_2['palId'] = "pal/10-2017/123456789/B"
+
+    # Fix PAL frequency for device_a.
+    pal_low_frequency = 3550000000
+    pal_high_frequency = 3560000000
+    ppa_record_1, pal_record_1 = makePpaAndPalRecordsConsistent(ppa_record_1,
+                                                            [pal_record_1],
+                                                            pal_low_frequency,
+                                                            pal_high_frequency,
+                                                            device_a['userId'])
+    # Move device_a into the first PPA zone.
+    device_a['installationParam']['latitude'], device_a['installationParam'][
+               'longitude'] = getRandomLatLongInPolygon(ppa_record_1)
+    device_a['userId'] = pal_record_1['licensee']
+
+    # Fix PAL frequency for device_b.
+    pal_low_frequency = 3600000000
+    pal_high_frequency = 3650000000
+    ppa_record_2, pal_record_2 = makePpaAndPalRecordsConsistent(ppa_record_2,
+                                                            [pal_record_2],
+                                                            pal_low_frequency,
+                                                            pal_high_frequency,
+                                                            device_a['userId'])
+    device_b['installationParam']['latitude'], device_b['installationParam'][
+               'longitude'] = getRandomLatLongInPolygon(ppa_record_2)
+    device_b['userId'] = pal_record_2['licensee']
+
+    config = {
+        'device_a': device_a,
+        'device_b': device_b,
+        'pal_record_1': pal_record_1,
+        'pal_record_2': pal_record_2
+
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_PCR_1_default_config)
+  def test_WINNF_FT_S_PCR_1(self, config_filename):
+    """ Successful Maximum PPA Creation
+
+    Checks PPA generated by SAS UUT shall be fully contained within the service area.
+    """
+    # Load the Config file
+    config = loadConfig(config_filename)
+    device_a = config['device_a']
+    device_b = config['device_b']
+    pal_record_1 = config['pal_record_1']
+    pal_record_2 = config['pal_record_2']
+
+    # Step 1: Inject one or more PAL records
+    self._sas_admin.InjectPalDatabaseRecord(pal_record_1[0])
+    self._sas_admin.InjectPalDatabaseRecord(pal_record_2[0])
+
+    # Step 2: Register device_a in SAS UUT
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    self._sas_admin.InjectUserId({'userId': device_a['userId']})
+    self._sas_admin.InjectFccId({'fccId': device_b['fccId']})
+    self._sas_admin.InjectUserId({'userId': device_b['userId']})
+
+    # Do registeration and check registration response of device_a and device_b in SAS UUT
+    cbsd_ids = self.assertRegistered([device_a, device_b])
+
+    # Step 3: Triggers SAS UUT to create a PPA boundary
+    palIds = [pal_record_1['palId']]
+    ppaIds = self._sas_admin.TriggerPpaCreation(cbsd_ids, palIds)
```

Comment:
This does not match the testing API that we have defined. Please review https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/sas_interface.py#L365